### PR TITLE
Completed TC03 31/8/2021

### PR DESCRIPTION
**Pull request by HoanVanHuynh**

### CODE REVIEW

File: src/test/java/tests/LoginTests.java [hunk 1]
```diff
@@ -49,4 +50,30 @@ public void tc02_CanNotLoginWithBlankUsername() {
...
         Assert.assertEquals(actualTitle, expectedTitle, "Title of login page should not be changed");
     }
+
+    @Test(description = "User cannot log into Railway with invalid password ")
+    public void tc03_CanNotLoginWithInvalidPassword() {
+        LogHelper.info("Click on Login tab");
+        loginPage.clickLoginTab();
+
+        LogHelper.info("Get invalid random password with 5 characters");
+        String invalidPassword = DataHelper.getRandomPassword(5);
```
Comment (tanmle) [inline src/test/java/tests/LoginTests.java]: we don't need to random an invalid password

Comment (HoanVanHuynh) [reply src/test/java/tests/LoginTests.java]: thank you, I'll change that

File: src/test/java/tests/LoginTests.java [hunk 2]
```diff
@@ -49,4 +50,30 @@ public void tc02_CanNotLoginWithBlankUsername() {
...
+        String expectedMessage = "Invalid username or password. Please try again.";
+
+        LogHelper.info("Verify that error message at top of login form appears correctly after logging with invalid password");
+        Assert.assertEquals(actualMessage, expectedMessage, "Error message at top of login form does not appear correctly as design");
+
+        LogHelper.info("Get current page title");
+        String actualTitle = DriverHelper.getTitle();
+        String expectedTitle = "Safe Railway - Login";
+
+        LogHelper.info("Verify that if user can not login, user is still at the login page");
```
Comment (MinhQuan119) [inline src/test/java/tests/LoginTests.java]: Because of invalid password, user must stay at login page, so don't need using "if user cannot login"

Comment (HoanVanHuynh) [reply src/test/java/tests/LoginTests.java]: Thank you so much, I used to think about that like you, so I'll delete that assertion.

**Review by MinhQuan119 — APPROVED** [review]

**Review by tanmle — APPROVED** [review]

**Pull request merged**